pkgs/by-name/dd/ddsmt/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ddsmt";
+  version = "2.0.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "ddSMT";
+    hash = "sha256-nmhEG4sUmgpgRUduVTtwDLGPJVKx+dEaPb+KjFRwV2Q=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    gprof2dot
+    progressbar
+  ];
+
+  meta = with lib; {
+    description = "A delta debugger for SMT benchmarks in SMT-LIB v2";
+    homepage = "https://github.com/ddsmt/ddSMT";
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [];
```

Since the package needs a maintainer, I will volunteer myself :-)

```suggestion
    license = with lib.licenses; [ gpl3Plus ];
    maintainers = with lib.maintainers; [ AndersonTorres ];
```

---

> Since the package needs a maintainer, I will volunteer myself :-)

Thanks!